nodebuilder/core/config.go

```diff
@@ -19,13 +19,20 @@ type Config struct {
 	IP       string
 	RPCPort  string
 	GRPCPort string
+	// TLSPath specifies the directory path where the TLS certificates are stored.
+	// It should not include file names('cert.pem' and 'key.pem').
+	// If left empty, the client will be configured for an insecure (non-TLS) connection.
+	TLSPath string
+	// XTokenPath specifies the file path to the JSON file containing the X-Token for gRPC authentication.
```

Comment:
comments say this is path to the JSON file, but due to the use of the `xtoken` const, needs to be the path to the directory containing a file named xtoken.json